images/win/scripts/ImageHelpers/ChocoHelpers.ps1

```diff
@@ -2,32 +2,31 @@ function Install-Choco {
     [CmdletBinding()]
     param(
         [Parameter(Mandatory)]
-        [string]$install,
-        [int]$retries = 5
+        [string] $PackageName,
+        [string[]] $ArgumentList,
+        [int] $RetryCount = 20

```

Comment:
This seems like a lot of retries and my concern is how much it could add to find out that it indeed will fail (no matter retries?)  I would think after 5 tries, if it fails then it's never going to succeed.